exotic.py

```diff
@@ -140,53 +140,6 @@ def binner(arr, n, err=''):
         err = np.array([np.sqrt(1. / np.nansum(1. / (np.array(i) ** 2.))) for i in why])
         return arr, err
 
-
-# finds the planet line in the composite dictionary
-# returns -1 if its not there
-def findPlanetLineComp(planName, dataDictionary):
-    coun = 0
-    for line in dataDictionary:
-        if line['fpl_name'] == planName:
-            index = coun
-        coun = coun + 1
-    return index
-
-
-# finds the line number of the planetName in the confirmed dictionary
-# returns -1 if its not there
-def findPlanetLineConf(planName, dataDictionary):
-    coun = 0
-    index = -1
-    # account for mistakes in capitalization, spaces, and dashes
-    noSpaceP = planName.replace(" ", "")
-    noSpaceDashP = noSpaceP.replace("-", "")
-    noCapsSpaceP = noSpaceDashP.lower()
-    for line in dataDictionary:
-        exoPname = line['pl_name']
-        exoPnoSpace = exoPname.replace(" ", "")
-        exoPnoSpaceDash = exoPnoSpace.replace("-", "")
-        exoPnoSpaceLower = exoPnoSpaceDash.lower()
-        if noCapsSpaceP == exoPnoSpaceLower:
-            index = coun
-        coun = coun + 1
-    if index != -1:
-        return index
-    else:
-        # print('Error! Could Not Find Planet with the name: '+planName)
-        # sys.exit()
-        return -1
-
-
-def findPlanetLinesExt(planName, dataDictionary):
-    coun = 0
-    indexList = []  # this has multiple lines per planet so list needed
-    for line in dataDictionary:
-        if line['mpl_name'] == planName:
-            indexList.append(coun)
-        coun = coun + 1
-    return indexList
-
-
```

Comment:
This seems to be the old way of searching for an exoplanet name in the NEA - removed it as we updated the API.